new/PRS13222SahlaSe.xml

```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS13222SahlaSe" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Śāhla Śǝllāse</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="MV"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="MV" when="2020-10-27">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ሣህለ፡ ሥላሴ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Śāhla Śǝllāse</persName>
+                    <faith type="EOTC"/>
+                    <floruit notBefore="1850" notAfter="1994"/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS13222SahlaSe" passive="MNC017">
+                        <desc>
+                            His name is written in a secondary hand in the supplication formulas throughout the manuscript, after the phrase <foreign xml:lang="gez">ገብርከ፡</foreign> 
+                            'your servant'. He might have been a secondary owner or donor of the manuscript. 
```

Comment:
When do we use patron?

Reply:
Well, since that name is written in a secondary hand, the individual at issue may have owned or donated the manuscript at some point of the Ms history, but probably not commissioned its writing.
If the name of the individual was written in the main hand, then "patron" would have well be included. What do you think?

Reply:
Okay, perfect. Thank you! I was just not sure, that we use "patron" only for "commissioner".